src/number.rs

```diff
@@ -1,7 +1,48 @@
 // Exports of <CoreFoundation/CFNumber.h>
 
+use libc::c_void;
+
 use base::*;
 
+#[repr(C)]
+struct __CFBoolean;
+pub type CFBooleanRef = __CFBoolean;
+
+extern "C" {
+    pub fn CFBooleanGetTypeID() -> CFTypeID;
+    pub fn CFBooleanGetValue(boolean: CFBooleanRef) -> Boolean;
+}
+
+pub type CFNumberType = CFTypeID;
```

Comment:
Where does `CFTypeID` come from? The header file I'm looking at defines `CFNumberType` as `CFIndex`:

```c
typedef CF_ENUM(CFIndex, CFNumberType) {
```